include/sound/soc-acpi.h

```diff
@@ -131,8 +131,8 @@ struct snd_soc_acpi_adr_device {
  */
 
 struct snd_soc_acpi_link_adr {
-	const u32 mask;
-	const u32 num_adr;
+	u32 mask;
+	u32 num_adr;
```

Comment:
nitpick: the commit message can mention that to achieve what is described, we will drop the `const` for some members.